content/learn/contribute/reference/triage.md

```diff
@@ -111,6 +111,8 @@ There are several paths for PRs to be closed:
 4. Anyone may nominate a PR for closure, by bringing it to the attention of the author and / or one of the SMEs / Maintainers. Let them press the button, but this is generally well-received and helpful.
 5. SMEs or Maintainers may, and are encouraged, to unilaterally close PRs that fall into one or more of the remaining categories.
 6. In the case of PRs where some members of the community, other than the author, are in favor, and some are opposed, any two relevant SMEs or Maintainers may act in concert to close the PR.
+7. For a PR that has been sitting for a while and became bitrotten, check with the original author if they intend to continue working on it. if not, or without a response, the PR can be labeled with `S-Adopt-Me`, and closed. Tracking adoption progress will happen in a linked issue.
```

Comment:
Seems like one of the issues with putting `S-Adopt-Me` on closed PRs is that there is no way to filter on PRs that _have not already been adopted or have been fully abandoned because the path was wrong_. Currently, someone looking to adopt a PR can just filter on open `S-Adopt-Me` PRs.

Should `S-Adopt-Me` also be added to the tracking issue to accommodate that? Put another way, should there be a way to filter issues based on whether or not they have adoptable PRs?

Reply:
Yup, I was thinking of adding the `S-Adopt-Me` label to the tracking issue too:
> *S-Adopt-Me* PRs should be closed, and a tracking issue opened with the same labels to track their adoptions.